treeherder/perf/utils.py

```diff
@@ -0,0 +1,31 @@
+import datetime
+
+MON, TUE, WED, THU, FRI, SAT, SUN = range(1, 8)
+
+
+def calculate_time_to_triage(created, due_days=3):
+    due_date = created
+
+    # if the alert was created in weekend, move the date to Monday
+    if due_date.isoweekday() == SAT:
+        due_date = due_date + datetime.timedelta(2)
+    if due_date.isoweekday() == SUN:
+        due_date = due_date + datetime.timedelta(1)
+
+    if due_date.isoweekday() + due_days < SAT:
+        # add the time-to days
+        due_date = due_date + datetime.timedelta(due_days)
+    # if the resulted date is on weekend, skip the weekend
+    elif due_date.isoweekday() + due_days in [SAT, SUN]:
+        due_date = due_date + datetime.timedelta(due_days + 2)
```

Comment:
this is technically incorrect as SAT needs +2, but SUN needs +1, I don't think it matters to be honest.

Reply:
this is made procedurally and i am checking every step of the calculation. if for example the date when the alert is created is SAT (6), we need to add 2 to move it to MON (6 + 2 = 8, otherwise 1 (SUN: 7 + 1 = 8).